pkg/pillar/scripts/device-steps.sh

```diff
@@ -177,6 +177,9 @@ wait_for_touch zedagent
 
 touch "$WATCHDOG_FILE/zedagent.touch"
 
+echo "$(date -Ins -u) Starting dmesg" >>$PERSISTDIR/log/dmesg.log
+dmesg -T -w --time-format iso >>$PERSISTDIR/log/dmesg.log &
+
```

Comment:
Looks like debug instrumentation forgotten to be removed.

Reply:
This code didn't belong in the PR. Thanks for catching it. Updated the PR.